lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -555,6 +557,8 @@ MITIGATION_02_CRI_DCMAW:
       targetState: PYI_ANOTHER_WAY
     fail-with-no-ci:
       targetState: PYI_ANOTHER_WAY
+    mitigation02:
+      targetState: PYI_ANOTHER_WAY
```

Comment:
Are there any other CRI states that could return `mitigation02` for the same reason?

I don't think so, because no other CRI should be used with a mitigation journey for now. When we add F2F I'm keen to normalise this and make sure we cover all the possible events in all cases, but it should be alright for now.

Reply:
To return `mitigation02` the user would have to have the particular CI (unmitigated)we're interested in, and not be initiating a new journey, so no, shouldn't be.